rotkehlchen/chain/manager.py

```diff
@@ -618,6 +620,9 @@ def query_balances(
         - EthSyncError if querying the token balances through a provided ethereum
         client and the chain is not synced
         """
+        self.totals = BalanceSheet()
```

Comment:
No no no no, absolutely not.

Do you have any idea what you are doing?

No matter the arguments, no matter which blockchain you query, no matter if you just want to query cache, this will clear up all the saved balances of all the blockchains.

Why the hell do you even touch this code?

Reply:
This was my terrible to attempt to quickly fix the problem that after account removal balances were kept. Surely will undo this change and will subtract balances on account removal